tests/ecosystem/upgrade/test_upgrade_ocp.py

```diff
@@ -0,0 +1,164 @@
+import logging
+import json
+
+from ocs_ci.framework import Config
+from ocs_ci.ocs import ocp
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework.testlib import ManageTest
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.exceptions import CephHealthException
+
+logger = logging.getLogger(__name__)
+
+# TODO: add image type validation (ga to ga , nightly to nightly, newer than current etc.)
+
+
+class TestUpgradeOCP(ManageTest):
+    """
+    1. check cluster health
+    2. check OCP version
+    3. perform OCP upgrade
+    4. check all OCP ClusterOperators
+    5. check OCP version
+    5. monitor cluster health
+    """
+    ocp_o = ocp.OCP()
+
+    def test_upgrade_ocp(self):
+        """
+
+        Tests OCS stability when upgrading OCP
+
+        """
+        config = Config()
```

Comment:
This will create an entirely different configuration object instead of using the configuration shared by the rest of the framework. Is this what you want or did you mean to import `config` from `framework` directly?